<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateTimeInfo</class>
 <widget class="QWidget" name="DateTimeInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="dateLabel">
     <property name="text">
      <string>Дата</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="timeLabel">
     <property name="text">
      <string>Время</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="dayOfWeekLabel">
     <property name="text">
      <string>День недели</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="deleteButton">
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
